src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java

```diff
@@ -45,7 +53,11 @@
                 new FeedforwardConstraint(3.0, FEEDFORWARD_CONSTANTS.getVelocityConstant(),
                         FEEDFORWARD_CONSTANTS.getAccelerationConstant(), false), // old value was 11.0
                 new MaxAccelerationConstraint(3.0), // old value was 12.5 * 12.0
-                new CentripetalAccelerationConstraint(3.0) // old value was 15 * 12.0
+                new CentripetalAccelerationConstraint(3.0), // old value was 15 * 12.0
+                //in inches
```

Comment:
For consistency, we should probably convert these values to meters (or convert the values in meters to inches).